packages/odyssey-react-mui/src/OdysseyThemeProvider.tsx

```diff
@@ -13,19 +13,23 @@
 import { ThemeProvider as MuiThemeProvider } from "@mui/material/styles";
 import { memo, ReactElement, useMemo } from "react";
 
-import { createTheme, deepmerge, ThemeOptions } from ".";
-import { odysseyTheme } from "./theme";
+import { createTheme, deepmerge, ThemeOptions, TokenOverrideOptions } from ".";
+import { createOdysseyTheme } from "./theme";
 
 const OdysseyThemeProvider = ({
   children,
   themeOverride,
+  tokenOverride,
```

Comment:
My 2 cents, based on what was asked last week during my rotation. Should we expose this prop via `OdysseyProvider` as well?

Reply:
I don't have any issue with it -- my goal here is whatever causes the least friction for downstream teams.

Reply:
Yep. I want to make `OdysseyProvider` (formerly `ThemeProvider`) have `tokenOverride` on it.